variables.tf

```
@@ -6,15 +6,20 @@ variable "cloudwatch_rule" {
       description   = string
       event_pattern = string
     })
-    event_target = object({
-      event_name       = string
-      event_target_id  = optional(string)
-      event_target_arn = string
-      role_arn         = string
-      input_transformer = object({
-        input_paths = map(string)
-      })
-      input_template = string
-    })
+    event_target = list(object({
+      rule           = string
+      target_id      = optional(string)
+      arn            = string
+      role_arn       = string
+      event_bus_name = optional(string)
+      dead_letter_config = optional(object({
+        arn = string
+      }))
+      input_transformer = optional(object({
+        input_paths    = optional(map(string))
+        input_template = string
+      }))
+
+    }))
   }))
 }
```

Here is the variable definition that would line up with my expectation of module usage:

```suggestion
variable "cloudwatch_rule" {
  description = "Object of input configs for cloudwatch event rules"
  type = object({
    event_rule = object({
      name          = string
      description   = optional(string)
      event_pattern = optional(string)
    })
    event_target = optional(list(object({
      target_arn = string
      target_id  = optional(string)
      role_arn         = optional(string)
      input_transformer = optional(object({
        input_paths = map(string)
      }))
      input_template = optional(string)
    })), [])
  }))
}
```